bindata/network/multus/multus.yaml

```diff
@@ -521,49 +524,63 @@ spec:
             defaultMode: 0744
 {{if .RenderIpReconciler}}
 ---
-apiVersion: batch/v1
-kind: CronJob
+apiVersion: apps/v1
+kind: Deployment
 metadata:
-  name: ip-reconciler
+  name: whereabouts-controlloop-deployment
   namespace: openshift-multus
   labels:
     tier: node
     app: whereabouts
 spec:
-  # reconcile loop every 15 minutes, starting at the top of the hour
-  schedule: "*/15 * * * *"
-  concurrencyPolicy: Replace
-  successfulJobsHistoryLimit: 0
-  jobTemplate:
+  selector:
+    matchLabels:
+      app: whereabouts
+  template:
+    metadata:
+      labels:
+        app: whereabouts
     spec:
-      backoffLimit: 0
-      ttlSecondsAfterFinished: 900
-      template:
-        spec:
-          priorityClassName: "system-cluster-critical"
-          serviceAccountName: multus
-          hostNetwork: true
-          containers:
-            - name: whereabouts
-              image: {{.WhereaboutsImage}}
-              resources:
-                requests:
-                  cpu: "25m"
-                  memory: "25Mi"
-              command:
-              - /ip-reconciler
-              - -log-level=verbose
-              volumeMounts:
-                - name: cni-net-dir
-                  mountPath: /host/etc/cni/net.d
-              env:
-              - name: KUBERNETES_SERVICE_PORT
-                value: "{{.KUBERNETES_SERVICE_PORT}}"
-              - name: KUBERNETES_SERVICE_HOST
-                value: "{{.KUBERNETES_SERVICE_HOST}}"
-          volumes:
-            - name: cni-net-dir
-              hostPath:
-                path: {{ .SystemCNIConfDir }}
-          restartPolicy: Never
+      containers:
+        - name: whereabouts-controlloop-deployment
+          securityContext:
+            privileged: true
+          image: "nicklesimba/whereabouts:prometheus-metrics"
+          resources:
+            requests:
+              cpu: "25m"
+              memory: "25Mi"
+          command:
+          - ./ip-control-loop
+          - -log-level=verbose
+          ports:
+          - name: metrics-port
+            containerPort: 1984
+          volumeMounts:
+          - name: system-cni-dir
+            mountPath: /host/etc/cni/net.d/whereabouts.d/
+        - name: kube-rbac-proxy
+          image: quay.io/coreos/kube-rbac-proxy:v0.5.0
+          args:
+            - --logtostderr
+            - --secure-listen-address=:8443
+            - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+            - --upstream=http://127.0.0.1:1984/
+            - --tls-private-key-file=/etc/metrics/tls.k
```

Comment:
typo .key is missing 
- --tls-private-key-file=/etc/metrics/tls.key

Reply:
Do I need the following line as well?

```- --tls-cert-file=/etc/metrics/tls.crt```